tfx_addons/firebase_publisher/runner.py

```diff
@@ -0,0 +1,213 @@
+"""Firebase Publisher runner module.
+
+This module handles the workflow to publish
+machine learning model to Firebase ML.
+"""
+import glob
+import os
+import tempfile
+from typing import Any, Dict, List, Tuple
+
+import firebase_admin
+import tensorflow as tf
+from absl import logging
+from firebase_admin import credentials, ml
+from firebase_admin.ml import ListModelsPage, TFLiteModelSource
+from tfx.dsl.io import fileio
+from tfx.utils import io_utils
+
+_SIZE_LIMIT_MB = 80
+
+
+def prepare_fb_download_model(app_name: str, credential_path: str,
```

Comment:
nit: If the functions are not to be used outside of here, lets make this functions private (aka append _ before function name). This makes sure users dont rely on this code externally and allows us to modify it without breaking backwards compatibility.